docs/README.adoc

```diff
@@ -29,6 +29,7 @@ The link above is for the doxygen docs on master. Doxygen docs for the following
 * https://advancedtelematic.github.io/aktualizr/2019.6/index.html[2019.6]
 * https://advancedtelematic.github.io/aktualizr/2019.7/index.html[2019.7]
 * https://advancedtelematic.github.io/aktualizr/2019.8/index.html[2019.8]
+* https://advancedtelematic.github.io/aktualizr/2019.8/index.html[2019.9]
```

Comment:
```suggestion
* https://advancedtelematic.github.io/aktualizr/2019.9/index.html[2019.9]
```

Reply:
This is not the first time I've stupidly done this! Thanks for catching it.